docs/UserGuide.md

```diff
@@ -262,50 +289,120 @@
 
 <div class="content">
 
-<strong>1. </strong>Ensure you have [Java `17`](https://www.oracle.com/java/technologies/downloads/#java17) installed in your Computer. Higher versions may work but we do not officially support it.
+<strong>1. </strong>**Download** and **install** [Java `17`](https://www.oracle.com/java/technologies/downloads/#java17) if you don't have it installed. We don't support other versions.
+
```

Comment:
should we suggest that it may work but we do not officially support it? Stating that we do not support other versions makes it sound like it doesn't work at all at higher versions but it still may work.